app/models/game.rb

```diff
@@ -36,4 +36,27 @@ def other_player(player)
   def pass_turn(player)
     update_attributes(active_player_id: other_player(player))
   end
+
+  def stalemate?
+    king_not_in_check? && available_move?
```

Comment:
Sean i think you're right, because a stalemate occurs when the King is NOT in check (king_not_in_check?) AND has no legal move to play, meaning black cannot play, game ends --> stalemate.

So to reiterate I also think that the logic should be:
`king_not_in_check? && !available_move?`